spec/javascripts/ActiveDocs/ActiveDocsSpec.spec.jsx

```diff
@@ -16,7 +16,5 @@ it('should render itself', () => {
 
 it('should render swagger-ui', () => {
   const wrapper = mount(<ActiveDocsSpec url={url} />)
-  console.log(wrapper.debug())
-
   expect(wrapper).toMatchSnapshot()
```

Comment:
Instead of adding the snapshot, since the rendered result it not very meaningful, I would play with `url`:
```suggestion
  expect(container.querySelector(`[url="${url}"]`)).toBeInTheDocument()
```

Reply:
We are not using `@testing-library/react` in `porta`, just in `portafly`

Reply:
Well even better, since we're not limited to `container.query`.
```suggestion
  expect(wrapper.find(SwaggerUI).hasProp('url')).toBe(true)
```
I think the only thing we should care about is the actual specification is rendered. I think we can _find_ the methods and resources from our spec. The problem is `foo.example.com/api_docs/account_data.json` is an empty file so we would need to mock it somehow.

But what is clear to me is that the snapshot doesn't really count as a test.